plugins/lando-recipes/lamp/lamp.js

```diff
@@ -336,6 +337,36 @@ module.exports = function(lando) {
     build.services = services(config);
     build.tooling = tooling(config);
 
+    // Determine the service to run cli things on
+    var unsupportedCli = (config.php === '5.3' || config.php === 5.3);
+    var cliService = (unsupportedCli) ? 'appserver_cli' : 'appserver';
+
+    // Build an additional cli container if we are running unsupported
+    if (unsupportedCli) {
+
+      // Build out a CLI container and modify as appropriate
+      var cliImage = 'devwithlando/pantheon-appserver:5.5-fpm';
+      build.services[cliService] = _.cloneDeep(build.services.appserver);
+      build.services[cliService].type = 'php:5.5';
+      build.services[cliService].via = 'cli';
+      build.services[cliService].overrides.services.image = cliImage;
+
+      // Remove stuff from appserver
+      delete build.services.appserver.build;
+
+    }
+
+    // Check if the user specified the compserSwitch key to false
+    var disableComposer = _.get(config, 'disableAutoComposerInstall', false);
```

Comment:
@serundeputy so i think we want this stuff to exist directly in the php service. the only thing the lamp recipe should be doing is checking for the `disableAutoComposerInstall ` setting in its `config` and then passing that down to the php service just like we do for the php version.

Reply:
I'm not sure how to do this one ... the php service `plugins/lando-services/php/php.js` doesn't have a 

```js
var build = function(...
```
step.

Should I add one?

Reply:
im not sure it needs `var build`, i think you probably just want to do what our composer key does
https://github.com/lando/lando/blob/master/plugins/lando-services/php/php.js#L332

Reply:
@serundeputy so i think it should go in the `services` function and work similarly to how we handle the `composer` global deps key eg: check for something (existence of `composer.json` in `app.root`) and then add a `composer install` to `config.build`